app/src/main/res/layout/widget_qr_view.xml

```diff
@@ -78,27 +78,15 @@
         app:layout_constraintTop_toBottomOf="@id/qrFrame"
         tools:text="5faZRizz9rm8S9tampULb2G6JwcdJPoQ1DK4bdRAzbXz" />
 
-    <ImageView
-        android:id="@+id/logoImageView"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="16dp"
-        android:paddingBottom="16dp"
-        android:src="@drawable/ic_p2p_validator"
-        android:visibility="gone"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/valueTextView" />
-
```

Comment:
You had to change logo here, not to remove it, I guess

Reply:
agree we need KeyApp here